capstone/src/main/java/com/google/sps/data/Lesson.java

```diff
@@ -78,4 +94,12 @@ public void setDescription(String description) {
     public Date getDate() {
         return (Date) this.entity.getProperty(Lesson.DATE_PROPERTY_KEY);
     }
+
+    public Entity getLessonEntity() {
```

Comment:
(optional) without any other entity getters, this might as well be called getEntity since we already know it is a lesson from its class.

Reply:
Sounds good. I think for now, I will leave it since many of the other class has a getUserEntity or getRoomEntity. This goes a while back where I suggested we have an interface and anything that has an entity as its instance variable, it could implement getEntity().

Reply:
Maybe in a later PR, we can change the names of all of the getter methods for the other entities.

Reply:
Yep, grouping all of these together into one PR as opposed to piecemeal is a great idea (if we have time)